dot/types/babe_digest.go

```diff
@@ -51,20 +52,6 @@ func NewBabePrimaryPreDigest(authorityIndex uint32,
 	}
 }
 
-// ToPreRuntimeDigest returns the BabePrimaryPreDigest as a PreRuntimeDigest
-func (d *BabePrimaryPreDigest) ToPreRuntimeDigest() (*PreRuntimeDigest, error) {
-	digest := NewBabeDigest()
-	err := digest.Set(*d)
-	if err != nil {
-		return nil, err
-	}
-	enc, err := scale.Marshal(digest)
-	if err != nil {
-		return nil, err
-	}
-	return NewBABEPreRuntimeDigest(enc), nil
-}
```

Comment:
why are these deleted? curious why you favour a standalone function over a method on the digests

Reply:
All `ToPreRuntimeDigest` methods use the same code. This is only to reduce repetitive code. That change deliberately has a commit of its own, so it can be easily reverted, if you prefer that.

Reply:
I like @qdm12 's suggestion!

Reply:
Made this change